index.html

```diff
@@ -5224,6 +5224,79 @@ <h2>IANA Considerations</h2>
         </li>
       </ul>
 
+<section id="media-type">
+  <h2>The <code>application/credential+json</code> Media Type</h2>
```

Comment:
I don't think I can accept this change, because the intention is to define the media type for the "data model" not the "secured data model".... unless we say that a "verifiable credential" can be "unsecured" similar to how "alg: none" is used... If thats ok, I am happy to accept this change, see the proposals on the mailing lists for the intended use of this `cty` with COSE and JOSE.

Reply:
definitions of the JOSE header parameters are very clear that

`cty` which is used to [`declare the media type of the secured content (the payload)`](https://www.rfc-editor.org/rfc/rfc7515.html#section-4.1.10) , should be `credential+xxx` and not `verifiable-credential+xxx`.

`typ` which is used to [`declare the media type of this complete JWS`](https://www.rfc-editor.org/rfc/rfc7515.html#section-4.1.9) should be `verifiable-credential+xxx`.

the ship that vc-data-model spec defined `credential` in a different way than identity industry used to define/use it has sailed. with `credential` term being more and more accepted as it is defined in vc-data-model spec, with good arguments, registration of `credential+xxx` should be possible.

Reply:
> with credential term being more and more accepted as it is defined in vc-data-model spec, with good arguments, registration of credential+xxx should be possible.

Alright, if the VCWG is going to get behind that argument, it might be doable. We'll have to see what the media types list says.